modules/specific-environment-permissions/environment_permissions.tf

```diff
@@ -85,6 +85,22 @@ resource "aws_iam_role_policy_attachment" "jenkins_role_lambda_attachment" {
   role       = aws_iam_role.jenkins_lambda_assume_role.name
 }
 
+
+resource "aws_iam_role" "jenkins_run_ssm_role" {
+  name               = "TDRJenkinsRunSsmRole${local.env_title_case}"
+  assume_role_policy = templatefile("${path.module}/templates/ecs_assume_role_policy.json.tpl", {})
+}
+
+resource "aws_iam_policy" "tdr_jenkins_run_ssm_policy" {
```

Comment:
Is the "tdr_" prefix needed for the resource name? Not used elsewhere.

Reply:
True, I've removed it.